go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -128,8 +146,8 @@ func fetchKnownCells(ctx context.Context, topoServer *topo.Server, target *query
 //	}
 //
 //	// Checking whether to throttle can be done as follows before starting a transaction.
-//	if t.Throttle() {
-//	  return fmt.Errorf("Transaction throttled!")
+//	if err := t.Throttle(); err != nil {
```

Comment:
Let's indent this entire code sample by a few tabs. Right now it makes the appearance that it's actual code being commented. If we indent it enough that it will become clear this is a code sample as part of the comment.